index.js

```diff
@@ -54,18 +68,204 @@ client.on("messageReactionAdd", async (reaction, user) => {
     }
 
     // Check if reaction emoji matches first target emoji:
-    if (reaction.emoji.name === firstTargetEmoji) {
-        emojiReactToSend(reaction.message, user, firstTargetChannelId);
+    if (reaction.emoji.name === PIN_TARGET_EMOJI) {
```

Comment:
Could refactor this to use a JS object that maps each emoji to the channel ID instead

Reply:
Will do in a followup PR